toolkit/tools/pkg/imagecustomizerlib/main_test.go

```diff
@@ -35,10 +34,8 @@ const (
 )
 
 var (
-	baseImageCoreEfiAzl2    = flag.String("base-image-core-efi-azl2", "", "A core-efi 2.0 image to use as a base image.")
-	baseImageCoreEfiAzl3    = flag.String("base-image-core-efi-azl3", "", "A core-efi 3.0 image to use as a base image.")
-	baseImageCoreLegacyAzl2 = flag.String("base-image-core-legacy-azl2", "", "A core-legacy 2.0 image to use as a base image.")
```

Comment:
Are we okay not testing the legacy images we produce from our build pipeline?
Theoretically, uefi->legacy->legacy->uefi should cover it. But I'd vote for testing whatever that is dropped in our artifacts as well (just in case - who knows what differences might be there).

Reply:
I'd step back and ask what should come out of the seeder / imagerv2. Should that always build uefi, and then any one who wants to build legacy goes through MIC?

That way - we'd have one code.

Reply:
Yeah, there is the possibility for differences, at least until image customizer takes over the building of the variant images. But legacy boot is kind of a niche scenario which ideally no-one should be using anymore. So, I am not sure it is worth keeping the complexity of accepting legacy images as input.